### ISSUE TITLE
c10::Error: Couldn’t find an operator for aten::dropout(Tensor input, float p, bool train)

### ISSUE THREAD

**Issue by un-knight** [pytorch/pytorch#43488]
I got an error with `c10::Error: Couldn't find an operator for aten::dropout(Tensor input, float p, bool train)` when I tried to load a torchscript model by libtorch c++ api. The linking libtorch library is compiled with `script/build_pytorch_android.sh` script.
```
target platform: android arm64-v8a
pytorch tag: v1.6.0
```

And if I linked the program with x86 prebuilt libtorch library, and run it in x86 platform, all worked well.

**Comment by ngimel** [comment]
Probably due to selective mobile build that does not include dropout. Note, in inference you should not need dropout.

**Comment by un-knight** [comment]
> Probably due to selective mobile build that does not include dropout. Note, in inference you should not need dropout.

Thanks, but how can I turn on dropout op support during the building, because I really need dropout op in inference in my case.

**Comment by dhruvbird** [comment]
@un-knight If you are using custom build as mentioned in https://pytorch.org/mobile/android/#custom-build then you can manually add the operator name in the generated YAML file before building. Are you using custom build in the first place?

**Comment by un-knight** [comment]
> @un-knight If you are using custom build as mentioned in https://pytorch.org/mobile/android/#custom-build then you can manually add the operator name in the generated YAML file before building. Are you using custom build in the first place?

@dhruvbird  Thanks for your reply, I had tried the [custom-build](https://pytorch.org/mobile/android/#custom-build) tutorial, and everything goes well for the compiling part, but meet `c10::Error: Couldn't find an operator for aten::dropout(Tensor input, float p, bool train)` error when running program in android adb environment. Besides, the dropout op is a standard pytorch op, so I think there is not need to custom build for dropout.
